Assets/Scripts/UI/SliderFocus.cs

```diff
@@ -0,0 +1,25 @@
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+/// <summary>
+/// This class must be attached to every slider, It reselects the 
+/// slider element after the player selects it.
```

Comment:
Nice solution to the slider issue!

Reply:
I don't mind any of the comments, I'm still rather new to any kind of UI development, so every nitpick helps 👍. Most likely we'll do an entire new redesign of the UI once we have more models, textures, etc. available and a theme locked down for the game.

Reply:
Yeah. It's nice to have a base that doesn't have any glaring flaws. I suspect we will keep the settings menu quite close to its current layout though, since this is fairly standard.